tests/update/updates_packagekit_gpk.pm

```diff
@@ -43,8 +43,11 @@ sub tell_packagekit_to_quit {
 # Update with GNOME PackageKit Update Viewer
 sub run {
     my ($self) = @_;
-    # updates_packagekit_gpk is disabled for SLE15 because of bsc#1060844
-    return record_soft_failure 'bsc#1060844' if sle_version_at_least('15') && is_sle();
+    if (is_sle && sle_version_at_least('15')) {
+        select_console 'root-console';
+        zypper_call("in gnome-packagekit", timeout => 90);
+        record_soft_failure 'bsc#1081584';
```

Comment:
Actually this code is wrong because the zypper in call will just ignore if the package actually is there so the soft-failure is shown regardless which it shouldn't be. See http://lord.arch.suse.de/tests/610#step/updates_packagekit_gpk/11 for an example showing the bug even though gnome-packagekit is already installed as a package. please fix by checking the return values of the zypper_call